src/editor/template/.gitignore.template

```diff
@@ -0,0 +1,7 @@
+.godot/
+/android/
+.gradle
+.kotlin
+.idea
+build
+jre
```

Comment:
TBH i would move the templates to the build tools module and use that one from the IDE plugin and use path ref here (as you would execute the python script when our project is checked out anyways). or directly do a path ref to the template files in the IDE plugin project.

Now we have a duplication of the template files and we need to remember to update both

Reply:
I can't do that as it is
The Intellij templates are for the WHOLE godot project, not just the additional JVM files,  and they are all in a flat directory.
For theGodot editor plugin I just need a subset of those and on top of that the python script work recursively so I just set the template files in the same relative directories as the final output.
The intellij plugin doesn't work that way so as it is right now, it's not really compatible. 
We can unify it in a separate PR.